doc/docs/modules/ROOT/pages/kafka-ssl.adoc

```diff
@@ -195,5 +195,79 @@ kafka.sasl.mechanism=PLAIN
+
+== Authorization with ACL's
+
+When Kafka is configured to use ACL's, you have to add also the following properties on the `neo4j.conf` file:
+
+[source, properties]
+----
+kafka.authorizer.class.name=kafka.security.authorizer.AclAuthorizer
+kafka.zookeeper.set.acl=true
+----
+
+[NOTE]
+* `kafka.security.authorizer.AclAuthorizer` (the default Kafka authorizer implementation), was introduced in Apache Kafka 2.4/Confluent Platform 5.4.0. If you are running a previous version, then use SimpleAclAuthorizer (`kafka.security.auth.SimpleAclAuthorizer`). If you are using the Confluent platform, you can use also the LDAP authorizer (please refer to the official Confluent documentation for further details: https://docs.confluent.io/platform/current/security/ldap-authorizer/quickstart.html)
+* Please consider that `zookeeper.set.acl` is **false** by default
+
+From the official Kafka documentation you can find that if a resource has no associated ACLs, then no one is allowed to access that resource except super users.
+If this is the case in your Kafka cluster, then you have also to add the following:
+
+[source, properties]
+----
+kafka.allow.everyone.if.no.acl.found=true
+----
+
+[NOTE]
+Be very careful on using the above property because, as the property name implies, it will allow access to everyone if no acl were found
+
+If super users are specified, then include also:
+
+[source,properties]
+----
+kafka.super.users=...
+----
+
+Moreover, if you change the default user name (principal) mapping rule then you have to add also the following properties:
+
+* If you used SSL encryption, then:
+
++
+[source, properties]
+----
+kafka.ssl.principal.mapping.rules=...
+----
+
+* If you used SASL encryption (probably so, if you have Kerberos environment), then:
+
++
+[source, properties]
+----
+sasl.kerberos.principal.to.local.rules=...
+----
+
+Furthermore, if you want to ensure that also the brokers communicates with each other using Kerberos,
+you have to specify the following property, which is anyway not required for the ACLs purposes:
+
+[source, properties]
+----
+kafka.security.inter.broker.protocol=SASL_SSL
+----
+
+[NOTE]
+The last property is `PLAIN` by default
+
+To make the plugin works properly, the operations that has to be authorized for the Topic and Cluster resource types are:
+
+* **Write**, when you want to use the plugin as a Source
+* **Read**, when you want to use the plugin as a Sink
+* **DescribeConfigs** and **Describe**, because the plugin uses the following 2 Kafka AdminClient API:
+    ** listTopics
+    ** describeCluster
+
+About the Streams Procedures you don't need to authorize a particular operation because the producer and the consumer created for the Source and the Sink will be reused by the procedures. This means that if you, for example, authorize the WRITE operation on a topic then you can use both the Source and the Procedure, depending on what you prefer.
```

Comment:
Revise:   "To use streams procedures, the same operations must be authorized (read or write) depending on which of the procedures you wish to use.  The permissions required by the procedures and the source/sink operations are the same."